gatsby-node.esm.js

```diff
@@ -291,6 +299,14 @@ exports.sourceNodes = async ({ actions, getNodes }) => {
             ? dataForCurrentContest[0].codeAccess
             : undefined,
       });
+      createNodeField({
+        node,
+        name: `type`,
+        value:
+          dataForCurrentContest.length > 0
+            ? dataForCurrentContest[0].type
+            : undefined,
+      });
```

Comment:
As the type is defined as `type: String` in the schema, I think we can't return undefined here.